crates/context_server/src/manager.rs

```diff
@@ -233,11 +244,18 @@ impl ContextServerManager {
         for (id, factory) in
             registry.read_with(&cx, |registry, _| registry.context_server_factories())?
         {
-            let config = desired_servers.entry(id).or_default();
-            if config.command.is_none() {
-                if let Some(extension_command) = factory(project.clone(), &cx).await.log_err() {
-                    config.command = Some(extension_command);
+            let config = desired_servers.entry(id.clone()).or_default();
+            match config {
+                ServerConfig::Stdio { command, .. } => {
+                    if command.is_none() {
+                        if let Some(extension_command) =
+                            factory(project.clone(), &cx).await.log_err()
+                        {
+                            *command = Some(extension_command);
+                        }
+                    }
                 }
+                ServerConfig::Sse { .. } => {}
```

Comment:
I'm not 100% sure what we are trying to do in this section, so I simply implemented a no-op, but I'm sure there's more to it... 😬